database-mongodb/usersSeed.js

```diff
@@ -0,0 +1,113 @@
+// SEEDING SCRIPT
+// "db:usersSeed": "node database-mongodb/usersSeed.js"
+
+const faker = require('faker');
+const db = require('./index.js');
+const User = require('./user.js');
+
+const sampleUsers = [];
+
+function getRandomArbitrary(min, max) {
```

Comment:
Seems like we have two random number generation functions, is it possible to use only one?

Reply:
Fixed, thanks